pkg/filters/int.go

```diff
@@ -28,8 +33,8 @@ func newIntFilter(is32Bit bool) *IntFilter {
 	return &IntFilter{
 		Equal:    []int64{},
 		NotEqual: []int64{},
-		Greater:  GreaterNotSetInt,
-		Less:     LessNotSetInt,
+		Greater:  maxIntVal,
+		Less:     minIntVal,
```

Comment:
In BPF code we now call Greater(Than) Min (to signify that this is the minimal allowed value), and Less(LowerThan) Max. I think we should use the same terminology here to avoid confusions.

Reply:
I export these as `Minimum()` and `Maximum()` for usage in the ebpf context, in the context of the internal logic of the filter I find it clearer with LessThan and GreaterThan (because there's also the "ongoing" logic of setting these values while parsing, userspace filters have further context to them than ebpf).